cmd/buildDeployApplication.go

```diff
@@ -74,7 +74,7 @@ func init() {
 	if buildDeployApplicationEnabled {
 		createJobCmd.AddCommand(buildDeployApplicationCmd)
 		buildDeployApplicationCmd.Flags().StringP("application", "a", "", "Name of the application to build-deploy")
-		buildDeployApplicationCmd.Flags().StringP("branch", "b", "master", "Branch to build-deploy from")
+		buildDeployApplicationCmd.Flags().StringP("branch", "b", "main", "Branch to build-deploy from")
```

Comment:
Chaning default to "main" is a breaking change. I think we should keep the old default

Reply:
Reverted